uw_uwnetid/subscription_64.py

```diff
@@ -0,0 +1,21 @@
+"""
+Interface for interacting with the UWNetID Subscription 64
+"""
+
+from uw_uwnetid.models import Subscription
+from uw_uwnetid.subscription import get_netid_subscriptions
+
+
+subs_code = Subscription.SUBS_CODE_2FA
+
+
+def get_2fa_subs(netid):
```

Comment:
Does this method return None or raise a DataFailureException if a subscription is not found?

Reply:
yes. i updated the doc